src/test/regress/expected/citus_local_tables.out

```diff
@@ -238,7 +240,199 @@ ERROR:  cannot create foreign key constraint since relations are not colocated o
 ALTER TABLE citus_local_table_1 ADD CONSTRAINT fkey_c_to_local FOREIGN KEY(a) references local_table(a);
 ERROR:  cannot create foreign key constraint
 ALTER TABLE local_table ADD CONSTRAINT fkey_local_to_c FOREIGN KEY(a) references citus_local_table_1(a);
-ERROR:  relation local_table is not distributed
```

Comment:
this obviously shouldn't be, it is one the cases I will handle in utility command pr